tools/docker/nginx.apps.conf

```diff
@@ -22,8 +29,7 @@ server {
 
         location / {
           try_files $uri $uri/ /APP_NAME/index.html;
-
-          add_header Cache-Control 'max-age=86400'; # 24h
+          add_header Cache-Control "no-cache" always;
```

Comment:
I don't think this is used at all since `try_files` does a redirect on the other locations. We might as well get rid of it I guess?